_opensearch/search/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Searching data
+nav_order: 20
+has_children: true
+has_toc: false
+---
+
+# Searching data
+
+Expectations from search engines have evolved over the years. Just returning relevant results quickly is no longer enough for most users. OpenSearch includes many features that enhance the user’s search experience as follows:
```

Comment:
We should have another comparison statement after sentence two. What are your users looking for beyond relevant results? Maybe something like,

"Now, users seek methods for to help get even more relevant results, sort and organize results, and a way to highlight their queries. For a more modern search experience, OpenSearch includes many features that enhance the user's search experience as follows:"